repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/framework/mocks/MockIdentityService.groovy

```diff
@@ -241,7 +241,7 @@ class MockIdentityService {
                 }
             }
 
-        } else if (nonQueryPath.startsWith("/users/")) {
+        } else if (nonQueryPath.startsWith("/v2.0/users/")) {
```

Comment:
I've added /v2.0 to this mock in a few places because... well, I'm not sure why it wasn't there or how our other tests pass. I imagine this may cause other tests to fail. Ugh.